README.md

```diff
@@ -15,6 +15,13 @@ You may also be interested in the hosted TensorBoard solution at
 share your ML experiments for free. For example, [this experiment][] shows a
 working example featuring the scalar dashboard.
 
+TensorBoard can run entirely offline on your local machine, in a corporate
+environment, or in a datacenter, without connecting to Google servers. There is
+one exception: the optional profile plugin, when active, connects to a
+Google-hosted CDN to load the Google Charts library. Your data is never shared
```

Comment:
"Nevertheless, in these environments, none of your data is shared ..."

We want to be careful in defining what "your data" is. Do you think it's clear that an IP address isn't included in "your data"?